portal/gil/templates/gil/index.html

```diff
@@ -44,7 +44,7 @@
           <h3 class="module__head">{{_("What is Prostate Cancer?")}}</h3>
           <p class="module__body">{{_("Prostate cancer is the second most common cancer in men.")}}</p>
           <p class="module__body">{{_("1 in 9 men will be diagnosed with prostate cancer in their lifetime.")}}</p>
-          <p class="module__body">{{_("In 2015, approximately 220,800 men were diagnosed with prostate cancer in the USA.")}}</p>
+          <p class="module__body">{{_("In 2019, over 174,500 men will be diagnosed with prostate cancer in the USA.")}}</p>
```

Comment:
This might be better (less likely to need re-translation). I'd imagine they'd want to keep this up to date
```suggestion
          <p class="module__body">{{_("In %d, over %s men will be diagnosed with prostate cancer in the USA.", 2019, '174,500')}}</p>
```

Reply:
Nice!! I forgot you can use named variables for those :nerd_face: